packages/cli/doc-core/src/theme-default/logic/useEditLink.ts

```diff
@@ -1,14 +1,18 @@
 import { EditLink } from 'shared/types';
+import { usePageData } from '@/runtime';
+
+export function useEditLink(editLink?: EditLink) {
+  const pageData = usePageData();
```

Comment:
It is a little messy there. You have already call `usePageData` and access the site config, so you don't need to add the `editLink` param.

By the way, i hope the responsibility of `useEditLink` is to provider a complete repo url and the caller doesn't need to care about some other details.So the hook would have the following optimizations:

1. delete the `editLink` param
2. join the `docRepoBaseUrl` and `relativePagePath`, take the result as the edit link.Finally return `text` and `link`.

Reply:
good，I will try to modify it this way

Reply:
I have made modifications
![image](https://user-images.githubusercontent.com/49436413/235636046-eb10bfb8-28e1-4411-a7c5-58942e5ca529.png)